modules/apps/captcha/captcha-taglib/src/main/resources/META-INF/resources/captcha/recaptcha.jsp

```diff
@@ -14,6 +14,8 @@ String errorMessage = (String)request.getAttribute("liferay-captcha:captcha:erro
 <c:if test="<%= captchaEnabled %>">
 	<aui:script src='<%= HtmlUtil.escapeAttribute(captchaConfiguration.reCaptchaScriptURL()) + "?hl=" + HtmlUtil.escapeAttribute(locale.getLanguage()) %>' type="text/javascript"></aui:script>
 
+	<label class="hidden" for="g-recaptcha-response">Google Recaptcha</label>
```

Comment:
Hey @jgarcialr, Could you please put the "Google Recaptcha" raw string into some Language.properties and use the proper Language API so the message can be internationalized?

Thanks

Reply:
Hi @rafaprax 

Thanks for the review!! 

I have one doubt about this comment. Do you think it is really necessary to create an entry in Language.properties? They are two trademarked words and that in all languages will be the same words, it will not change (for example you can see [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_ja.properties#L79) that _Google Places API_ is not translated). I mean, we are going to ask for translations of trademarks, which are not translated. 

Thanks in advance!

Reply:
Hi @rafaprax ,

The label is hidden so it shouldn't be read by anyone, it just there to be compliant with the accesiibility tool.
The textarea is also hidden, so neither of those would be read by a screen reader for blind persons.

Reply:
Yeah @alvarosugarlr, Thanks to bring this up. I was talked to Jorge on DM and he explained that it is a hidden field and won't be read by an accessibility tool. So, let's keep as it is. 

Thanks guys